docker/configs/mariadb/initdb.sql

```diff
@@ -0,0 +1,6 @@
+ALTER DATABASE `bluerange` CHARACTER SET 'utf8mb4' COLLATE  'utf8mb4_general_ci';
+
+-- create grafana database
+CREATE DATABASE IF NOT EXISTS grafana_bluerange;
+CREATE USER 'grafana'@'%' IDENTIFIED BY '${DATABASE_PWD}';
```

Comment:
should we use a different password if only for protecting against accidental login to the wrong schema? consider adding a prefix/suffix so that this admittedly unlikely situ is prevented technically.